### DISCUSSION TITLE
Hide menu items for specific environment.

### DISCUSSION THREAD

**Discussion by sunnysonx** [strapi/strapi#6473] [Legacy Topics]
Hello,
I know you can hide some Content-types from menu using `admin/src/config/layout.js`, but how to deal with  Marketplace, Plugins, Settings and other sections in menu.
Basically I want to show some menu items if its the development build and hide them if its the production build. Any ideas are welcomed. 
Thank you!

**Comment by gonzarascon** [comment]
Hi! How did you hide Content-types from the menu? Is there a guide for that?

Have you tried `NODE_ENV` to dynamically show/hide elements based on enviroment?
